js/help.js

```diff
@@ -0,0 +1,49 @@
+const helpOverlay = new Image();
+helpOverlay.src = 'images/help-image.png';
+
+const helpBtn = document.querySelector('.help');
+
+let helpImage = false;
+
+function drawHelp() {
+  if (helpImage === true) {
```

Comment:
In industrial programming it would be right, but normally you can skip `===` here, writing just `if (helpImage) {}`